app/utils/requests/collaborationQuestions/requests.ts

```diff
@@ -67,8 +70,8 @@ export async function getBasicCollaborationQuestionById(id: string): Promise<Bas
 
 export async function getBasicCollaborationQuestionByIdWithAdditionalData(id: string) {
   try {
-    const response = await strapiGraphQLFetcher(GetCollaborationQuestionByIdQuery, { id });
-    const data = response.collaborationQuestion?.data;
+    const response = await strapiGraphQLFetcher(GetCollaborationQuestionByIdQuery, { documentId: id });
+    const data = response.collaborationQuestion as ResultOf<typeof CollaborationQuestionFragment>;
```

Comment:
cast is not necessary
```suggestion
    const data = response.collaborationQuestion
```

Reply:
Removed all ResultOf<...>  casting where it's not necessary.